nautobot/virtualization/forms.py

```diff
@@ -528,6 +543,21 @@ class VMInterfaceCreateForm(BootstrapMixin, InterfaceCommonForm):
     virtual_machine = DynamicModelChoiceField(queryset=VirtualMachine.objects.all())
     name_pattern = ExpandableNameField(label="Name")
     enabled = forms.BooleanField(required=False, initial=True)
+    parent_interface = DynamicModelChoiceField(
+        queryset=VMInterface.objects.all(),
+        required=False,
+        display_field="display",
+        query_params={
+            "virtual_machine_id": "$virtual_machine",
+        },
+    )
+    bridge = DynamicModelChoiceField(
+        queryset=VMInterface.objects.all(),
+        required=False,
```

Comment:
Same question here about `display_field` on `parent_interface` but not on `bridge`.